api/app/controllers/spree/api/v2/platform/menu_items_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class MenuItemsController < ResourceController
+          before_action -> { doorkeeper_authorize! :write, :admin }, only: WRITE_ACTIONS << :reposition
```

Comment:
@damianlegawiec I had to put this back in, for some reason moving `:reposition` to the `WRITE_ACTIONS` constant in the `ResourceController ` stopped the reposition action working, I'll look into that more and see if I can find out exactly why.